### PR TITLE
Add Akan - QX layout

### PR DESCRIPTION
**Pull request by amire80**
Made with Felix Nartey at Wikimania 2016.
Based on
http://www.kasahorow.org/node/260

### CODE REVIEW

File: rules/ak/ak-qx.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	'use strict';
+
+	var defs = {
+		id: 'ak-qx',
+		name: 'Akan - QX replacement',
+		description: 'Akan input method with Q and X replaced by Ɛ and Ɔ',
+		date: '2016-06-23',
+		URL: 'http://www.kasahorow.org/node/260',
+		author: 'Amir E. Aharoni, based on Kasahorow',
+		license: 'GPLv3',
```
Comment (kartikm) [inline rules/ak/ak-qx.js]: Just: GPL3

Comment (amire80) [inline rules/ak/ak-qx.js]: Is it according to any standard?
All other files use "GPLv3". Should it be replace everywhere? If yes, it should be done in another patch.
